src/OrbitGl/Batcher.cpp

```diff
@@ -44,11 +44,20 @@ void Batcher::AddVerticalLine(Vec2 pos, float size, float z, const Color& color,
   AddLine(pos, pos + Vec2(0, size), z, color, picking_color, nullptr);
 }
 
+static void MoveLineToPixelCenterIfHorizontal(Line& line) {
+  if (line.start_point[1] != line.end_point[1]) return;
+  line.start_point[1] += 0.5f;
+  line.end_point[1] += 0.5f;
+}
+
 void Batcher::AddLine(Vec2 from, Vec2 to, float z, const Color& color, const Color& picking_color,
                       std::unique_ptr<PickingUserData> user_data) {
   Line line;
+  // TODO(b/195386885) This is a hack to address the issue that some horizontal lines in the graph
+  // tracks are missing. We need a better solution for this issue.
   line.start_point = Vec3(floorf(from[0]), floorf(from[1]), z);
   line.end_point = Vec3(floorf(to[0]), floorf(to[1]), z);
+  MoveLineToPixelCenterIfHorizontal(line);
```

Comment:
Nit:
What do you think about this instead?
```
if (line.IsHorizontal()) {
  MoveLineToPixelCenter(line);
}
```

Reply:
Thanks for the suggestion! 

This is really a hack, so I want to put this into a single static free function rather than adding a new member function `Line::IsHorizontal` which is only used here.

And we only need to move horizontal lines to the center. The name of `MoveLineToPixelCenter` makes me feel like that there are cases that we need to do this also for vertical lines.